api/proto/message/message.proto

```diff
@@ -87,6 +87,7 @@ message ConsensusRequest {
   bytes block = 5;
   bytes sender_pubkey = 6;
   bytes payload = 7;
+  uint64 epoch = 8;
```

Comment:
why do you need the epoch here? Is there no other way to get epoch? We want to make consensus msg as light weight as possible because it's the majority of the traffic.

Reply:
Yes, I considered other ways. But unsure about how to best handle `ExplorerMessageHandler`.
 
Could add commit sig check after explorer can get a block from FBFTLog?

Reply:
Actually the signed content should always be consistent with those in the block. So you can always get the viewID from the block. Same for ExplorerMessageHandler